docs/faqs.md

```diff
@@ -91,6 +112,19 @@ Of course, on a specific platform if you have a reliable mechanism to detect if
 a volume is permanently deleted or cannot recover, you can write an operator or
 sidecar to automate this.
 
+2) Kubernetes node object has been recreated
+
+If by accident kubernetes node object has been recreated, new node object with
+same name for same node will get a new UID. By default, provisioner includes
+node.UID in its name, it will filter out local PVs that were not created with
+this name and do not reclaim these PVs. You can check
+`pv.kubernetes.io/provisioned-by` annotation in PV object.
+
+You can use enable `useNodeNameOnly` to set provisioner name without node UID
+to solve this cleanup issue. However, this will only fix new deployments going
+forward. Existing PVs created with an older local provisioner will still have
+the UID provisioner naming.
```

Comment:
https://github.com/kubernetes-incubator/external-storage/issues/945#issuecomment-414393266
Can we enable useNodeNameOnly by default in next release?

Reply:
We can enable it by default but probably need to cut a new major version